src/decima/interpret/modisco.py

```diff
@@ -0,0 +1,432 @@
+import warnings
+import logging
+from pathlib import Path
+from typing import List, Optional, Tuple, Union
+
+import numpy as np
+import modiscolite
+from tqdm import tqdm
+from more_itertools import chunked
+from torch.utils.data import DataLoader
+from grelu.resources import get_meme_file_path
+
+
+from decima.constants import DECIMA_CONTEXT_SIZE
+from decima.core.result import DecimaResult
+from decima.data.dataset import GeneDataset
+from decima.utils import get_compute_device
+from decima.utils.io import AttributionWriter
+from decima.core.attribution import AttributionResult
+from decima.interpret.attributer import DecimaAttributer
+
+
+def _get_on_off_tasks(result: DecimaResult, tasks: Optional[List[str]] = None, off_tasks: Optional[List[str]] = None):
+    if tasks is None:
+        tasks = result.cell_metadata.index.tolist()
+    elif isinstance(tasks, str):
+        tasks = result.query_cells(tasks)
+    if isinstance(off_tasks, str):
```

Comment:
So if off_tasks is None, it returns None - what happens if Specificity is supplied with off_tasks=None?